BZ-2049375:Adds command to Entitlement Build (#44472)
openshift/openshift-docs: modules/builds-source-secrets-entitlements.adoc

You must have access to Red Hat entitlements through your subscription, and the entitlements must have separate public and private key files.

[TIP]
====

> When performing an Entitlement Build using {op-system-base-full} 7 be sure to have the following in your Dockerfile before any `yum` commands:

Reviewer: A small tweak for second person perspective (preferred way per ISG), and to provide an [object reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references) for the Dockerfile command:
```suggestion
When you perform an Entitlement Build using {op-system-base-full} 7, you must have the following command in your Dockerfile before you run any `yum` commands:
```
The second suggestion about object references is from the supplemental guidelines, and it seems to just mean when you have a literal value within the sentence itself 🙂 but even in an indirect case like this one, I think it adds some clarity

> When you perform an Entitlement Build using {op-system-base-full} 7 you must have the following command in your Dockerfile before run any `yum` commands:

Reviewer 2: Still missing the comma after "7".

Also: s/before run/before you run/

> [source,terminal]

Reviewer 2: Only a suggestion: Technically this does render fine, but for source readability / consistency with most of the repo, consider a blank line before the code block (specifically, before the `[source,terminal]` in this case).

----

> $  oc RUN rm /etc/rhsm-host

Reviewer 2: Is the `$  oc` part supposed to be here? I thought this was intended to be a Dockerfile instruction.
